handshake_ix.go

```diff
@@ -73,206 +73,167 @@ func ixHandshakeStage0(f *Interface, vpnIp uint32, hostinfo *HostInfo) {
 
 }
 
+// Note we always return `false` in ixHandshakeStage1 because there is nothing
+// to tear down.
 func ixHandshakeStage1(f *Interface, addr *udpAddr, hostinfo *HostInfo, packet []byte, h *Header) bool {
```

Comment:
While we're cleaning up.... this hostinfo input value is never used (we always overwrite it now, but before your change we always shadowed it!), can we remove it?